lite-rpc/src/bridge.rs

```diff
@@ -178,7 +183,13 @@ impl LiteRpcServer for LiteBridge {
     }
 
     async fn get_cluster_nodes(&self) -> crate::rpc::Result<Vec<RpcContactInfo>> {
-        todo!()
+        Ok(self
+            .data_cache
+            .cluster_info
+            .cluster_nodes
+            .iter()
+            .map(|v| v.value().as_ref().clone())
```

Comment:
do not use RpcContactInfo type in store - the type is RPC- specific DTO

Reply:
We already store it in this form. There is no real reason to convert to any other form.